x-pack/plugins/lens/server/saved_objects.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { CoreSetup } from 'kibana/server';
+import { getEditPath } from '../common';
+
+export function setupSavedObjects(core: CoreSetup) {
+  core.savedObjects.registerType({
+    name: 'lens',
+    hidden: false,
+    namespaceAgnostic: false,
+    management: {
+      icon: 'lensApp',
+      defaultSearchField: 'title',
+      importableAndExportable: true,
+      getTitle: (obj: { attributes: { title: string } }) => obj.attributes.title,
+      getInAppUrl: (obj: { id: string }) => ({
+        path: getEditPath(obj.id),
+        uiCapabilitiesPath: 'visualize.show',
```

Comment:
:information_source: We used before this move `lens.show` here, which was wrong, since there is no `lens.show` uiCapabilities. We decided that time to use the visualize uiCapability for Lens, it seemed just this place always returned the wrong capability.